durablefunctionsmonitor.dotnetbackend/Common/Setup.cs

```diff
@@ -275,5 +283,23 @@ private static string GetVersion()
             var version = typeof(DfmEndpoint).Assembly.GetName().Version;
             return $"{version.Major}.{version.Minor}.{version.Build}";
         }
+
+        private static bool AreAppRoleListsIntersecting(params string[][] appRoleLists)
```

Comment:
Consider just

```
        private static bool AreAppRoleListsIntersecting(params string[][] appRoleLists)
        {
            try
            {
                appRoleLists.SelectMany(a => a).ToDictionary(a => a);

                return false;
            }
            catch (ArgumentException)
            {
                return true;
            }
        }
```

instead.
`.ToDictionary()` throws, if there's a key duplicate - and that's all we need here :)

Reply:
Thanks for the suggestion, that looks better. I just used a `Where` instead of the collection expression to make it compatible with dotnet 6 (durablefunctionsmonitor.dotnetbackend seems to use this version).